### ISSUE TITLE
Ubuntu 16.04 compatibility

### ISSUE THREAD

**Issue by liltaylor** [JuliaPy/Conda.jl#95]
I am trying to install PyPlot - all defaults - in Julia 6.1 in Ubuntu 16.04:
I simply Pkg.add("Conda"), Pkg.add("PyCall"), Pkg.add("PyPlot") and then 'using PyPlot'.

The crux is:
ImportError("/lib/x86_64-linux-gnu/libz.so.1: version `ZLIB_1.2.9' not found (required by /home/myusername/.julia/v0.6/Conda/deps/usr/lib/python2.7/site-packages/matplotlib/../../.././libpng16.so.16)",)

The error message tells the truth:
 the libpng16.so.16 installed with Conda depends on libz.so.1 version 1.2.9
my Ubuntu system, though, only has version 1.2.8
also, 1.2.8 is the version currently offered on apt

where do I go from here? I am at a loss.
I am wary of trying to install libz 1.2.9 system-wide unless it is available from ubuntu.

EDIT weary->wary (spelling!)

**Comment by isuruf** [comment]
Only way I can think of to fix this would be to do,
```
LD_LIBRARY_PATH="/home/myusername/.julia/v0.6/Conda/deps/usr/lib/" julia
```

**Comment by liltaylor** [comment]
It works! 
I cannot express how thankful I am!
I guess this solution is not long term (future missing libraries would need to be put in Conda's lib directory AFAICS)  but this made my day.
thanks again

**Comment by stevengj** [comment]
Fixed in Julia 0.6.3.
